services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py

```diff
@@ -428,15 +435,72 @@ async def pull_input_ports(
     return transferred_bytes
 
 
+@containers_router.patch(
+    "/containers/directory-watcher",
+    summary="Enable/disable directory-watcher event propagation",
+    response_class=Response,
+    status_code=status.HTTP_204_NO_CONTENT,
+)
+async def disable_directory_watcher(
+    patch_directory_watcher_item: PatchDirectoryWatcherItem,
+    app: FastAPI = Depends(get_application),
+) -> None:
+    if patch_directory_watcher_item.is_enabled:
+        directory_watcher.enable_directory_watcher(app)
+    else:
+        directory_watcher.disable_directory_watcher(app)
+
+
+@containers_router.post(
+    "/containers/ports/outputs/dirs",
+    summary=(
+        "Creates the output directories declared by the docker images's labels. "
+        "Since the dynamic-sidecar cannot retrieve the labels "
```

Comment:
one question? why is that?

Reply:
Because you would have to mount the docker socket in the dynamic-sidecar and if you are not an a manager node you do not have access to it.

Reply:
and you mean the docker socket is not mounted in the dynamic sidecar? how does it start a docker container then?

Reply:
> and you mean the docker socket is not mounted in the dynamic sidecar? how does it start a docker container then?

just relied this myself now. it is mounted.

Rethinking about it, I guess it was done for convenience. director-v2 already has all the machinery to call into director-v0 to fetch them. I guess I did not want to rewrite that piece of code.

Not sure if I should also put it here.

Reply:
well then at least change the comment since it is wrong.